### PR TITLE
#167165068 User can receive notifications

### PR DESCRIPTION
**Pull request by PelumiAlesh**
### What does this PR do?
Sends notification to the user
#### Description of Task to be completed?
User should be able to receive notifications when users I follow publish new articles.

User should be able to receive notifications when articles I favorited have new interactions (like new comments).

User should be able to opt in and out of these notifications.
#### Screenshots (if appropriate)

### CODE REVIEW

File: docs/index.js [hunk 1]
```diff
@@ -47,5 +49,8 @@ swagger.paths['/articles/{slug}/comments'] = postComment;
 swagger.paths['/articles/{slug}/like'] = likeArticle;
 swagger.paths['/articles/{slug}/dislike'] = dislikeArticle;
 
+swagger.paths['/notifications'] = getNotifcation;
+swagger.paths['notifications/{id}'] = updateNotification;
```
Comment (OdunayoOkebunmi) [inline docs/index.js]: Shouldn't there be a slash here?

Comment (PelumiAlesh) [reply docs/index.js]: Fixed thanks

File: controllers/NotificationController.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        },
+        {
+          where: {
+            id,
+            receiverId,
+          }
+        }
+      );
+      if (markRead[0]) {
+        return successResponse(res, 200, 'notification', 'Notifications successfully updated to read');
```
Comment (tolulope-od) [inline controllers/NotificationController.js]: please return your responses in an object so it can be uniform with other responses

**Review by tolulope-od — COMMENTED** [review]
Good job here Pelumi & Victor, please affect the changes pointed out, other than that this PR LGTM

**Review by OdunayoOkebunmi — APPROVED** [review]

**Review by tolulope-od — APPROVED** [review]

**Review by mekzy-o — APPROVED** [review]
LGTM

**Comment by mazma1** [comment]
Great work @PelumiAlesh and @VictorMaria. If the team says this is good to go, who am I? 😅

**Pull request merged**